### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by jlxw48**
* Correct grammar for `policy` command implementation
* Edit use cases for `policy` and `batch` commands
* Add user stories for `policy` and `batch` commands

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1040,13 +1022,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 1.  User requests to delete several clients at once.
 
-2.  ClientBook removes the client contacts from the list.
+2.  ClientBook removes the client contacts from the address book.
```
Comment (NBH99) [inline docs/DeveloperGuide.md]: The sentence could use the wording in delete use case: ClientBook deletes the clients.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -954,27 +958,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
 * 1a. The selected client has no policies.
 
-    * 1a1. Window launched shows that selected client has no associated policies.
+    * 1a1. ClientBook shows message indicating that no policies are associated with the selected client.
 
       Use case ends.
 
-* 1a. The given index is invalid.
+* 1b. The given arguments are invalid.
```
Comment (NBH99) [inline docs/DeveloperGuide.md]: It could be one or more of the given arguments are invalid.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -984,48 +988,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-      Use case resumes at step 1.
-
 <br>
 
 **Use case 18: Batch edit client details**
 
 **MSS**
 
-1.  User requests to change the policy ID of a policy shared by multiple customers.
+1.  User requests to change the details which are common to several clients.
```
Comment (NBH99) [inline docs/DeveloperGuide.md]: The original details could be different before using batch edit to change.

**Review by NBH99 — COMMENTED** [review]
Some minor problems, else LGTM after fixing.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jlxw48) [reply docs/DeveloperGuide.md]: What do you mean by this?

Comment (NBH99) [reply docs/DeveloperGuide.md]: The current sentence could mean that only when all the given arguments are invalid then it give error.

**Review by NBH99 — COMMENTED** [review]
LGTM

**Pull request merged**